### DISCUSSION TITLE
Remark-variables broken?

### DISCUSSION THREAD

**Discussion by celinehaddad** [remarkjs/remark#1271] [Q&A] [answered]
In [this list of plugins](https://github.com/remarkjs/remark/blob/main/doc/plugins.md#list-of-plugins) on the Remark GitHub, I see that remark-variables is marked as currently broken. Are there any updates on whether or not it can be used with `"remark": "^14.0.2"`?

**Comment by wooorm** [comment, marked as answer]
Hi Celine! It’s indeed not maintained. See the issue I raised there 3 years ago: https://github.com/mrzmmr/remark-variables/issues/3.

**Reply by celinehaddad** [reply]
Okay, thanks for the clarity and link @wooorm :)
